pkg/skaffold/runner/generate_pipeline.go

```diff
@@ -30,14 +30,21 @@ import (
 )
 
 func (r *SkaffoldRunner) GeneratePipeline(ctx context.Context, out io.Writer, config *latest.SkaffoldConfig, fileOut string) error {
+	// Keep track of files, configs, and profiles. This will be used to know which files to write
+	// profiles to and what flags to add to task commands
+	configFile := &pipeline.ConfigFile{
+		Name:    r.runCtx.Opts.ConfigurationFile,
```

Comment:
this actually represents the path. They might all be names `skaffold.yaml`
Considering `Name` to `Path` ?